bindata/assets/kube-controller-manager/svc-cluster-policy-controller.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-policy-controller
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: serving-cert-cpc
+    prometheus.io/scrape: "true"
+    prometheus.io/scheme: https
```

Comment:
Any reasons for having these annotations? They shouldn't be needed in OpenShift.

Reply:
Thanks, removed (wasn't sure about these).